app/assets/javascripts/admin/variant_overrides/controllers/variant_overrides_controller.js.coffee

```diff
@@ -13,6 +13,10 @@ angular.module("admin.variantOverrides").controller "AdminVariantOverridesCtrl",
   $scope.RequestMonitor = RequestMonitor
   $scope.selectView = Views.selectView
   $scope.currentView = -> Views.currentView
+  $scope.onDemandOptions = [
+    { description: t('js.variant_overrides.options.on_demand.unlimited'), value: true },
+    { description: t('js.variant_overrides.options.on_demand.limited'), value: false }
```

Comment:
I find that adding limited/unlimited to the on_demand concept is a bit confusing. Can we keep that complexity at the translations level and use something like this instead?
js.variant_overrides.options.on_demand.true and js.variant_overrides.options.on_demand.false
or js.variant_overrides.options.on_demand.active and js.variant_overrides.options.on_demand.inactive
These can still be translated to on_demand and Limited Stock.